win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -1082,22 +1108,28 @@ TEST_FUNCTION(chaos_knight_test_with_timers)
     for (i = 0; i < MAX_TIMER_COUNT; i++)
     {
         chaos_test_data.timers[i].timer = NULL;
-        InterlockedExchange(&chaos_test_data.timers[i].state, TIMER_STATE_NONE);
+        (void)InterlockedExchange(&chaos_test_data.timers[i].state, TIMER_STATE_NONE);
     }
 
     chaos_test_data.work_item_context = threadpool_create_work_item(chaos_test_data.threadpool, work_function, (void*)&chaos_test_data.executed_work_functions);
-
+    ASSERT_IS_NOT_NULL(chaos_test_data.work_item_context);
     for (i = 0; i < CHAOS_THREAD_COUNT; i++)
     {
-        thread_handles[i] = CreateThread(NULL, 0, chaos_thread_with_timers_func, &chaos_test_data, 0, NULL);
+        thread_handles[i] = CreateThread(NULL, 0, chaos_thread_with_timers_no_lock_func, &chaos_test_data, 0, NULL);
         ASSERT_IS_NOT_NULL(thread_handles[i], "thread %zu failed to start", i);
     }
 
     // wait for some time
-    Sleep(10000);
+    Sleep(TEST_RUN_TIME);
 
     (void)InterlockedExchange(&chaos_test_data.chaos_test_done, 1);
 
+    //will change to use InterlockedHL_waitforvalue_64 later, task:https://msazure.visualstudio.com/One/_workitems/edit/30197585
```

Comment:
>InterlockedHL_waitforvalue_64 [](http://example.com/codeflow?start=25&length=29)

Nitpick: InterlockedHL_WaitForValue_64 is likely the future API name

Reply:
added in the comment